src/components/Comments/Comment.js

```diff
@@ -240,15 +276,38 @@ const Comment = ({
         {isLast && level >= 2 ? (
           <View testID="level2" style={styles.gap} />
         ) : (
-          <TouchableOpacity activeOpacity={1} onPress={onPress} testID="memoComment">
-            <ButtonHightlight
-              onLongPress={handleOnLongPress}
-              style={styles.btnReply}
-              onPress={onPress}>
-              <MemoCommentReply />
-              <Text style={styles.btnReplyText}>Reply</Text>
-            </ButtonHightlight>
-          </TouchableOpacity>
+          <>
+            <TouchableOpacity activeOpacity={1} onPress={onPress} testID="memoComment">
+              <ButtonHightlight
+                onLongPress={handleOnLongPress}
+                style={[
+                  styles.btnReply,
+                  {
+                    marginRight: 0,
+                    paddingRight: 0
+                  }
+                ]}
+                onPress={onPress}>
+                <MemoCommentReply />
+              </ButtonHightlight>
+            </TouchableOpacity>
+            {!comment.is_you && (
+              <TouchableOpacity onPress={onPressDm} testID="sendDMbtn" activeOpacity={1}>
+                <ButtonHightlight
+                  onPress={onPressDm}
+                  style={[
+                    styles.btnBlock(comment.user.id === yourselfId),
+                    styles.btn,
+                    {
+                      marginRight: 0,
+                      paddingRight: 0
+                    }
+                  ]}>
+                  <MemoSendDM />
+                </ButtonHightlight>
+              </TouchableOpacity>
+            )}
+          </>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The UI for displaying the reply button and DM button has been updated. Ensure that these changes don't break the existing layout and are responsive across different screen sizes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->